src/components/input/FormField.tsx

```diff
@@ -1,9 +1,84 @@
 import { Info } from "@phosphor-icons/react";
 import type React from "react";
-import { useState } from "react";
-import { Input } from "./Input";
+import { forwardRef, useState } from "react";
+import { cn } from "@/lib/utils";
 
-interface FormFieldProps {
+// Internal Input component - only used by FormField
+type InternalInputProps = Omit<
+  React.InputHTMLAttributes<HTMLInputElement>,
+  "size"
+> & {
+  children?: React.ReactNode;
+  className?: string;
+  displayContent?: "items-first" | "items-last";
+  isValid?: boolean;
+  onValueChange?: (value: string, isValid: boolean) => void;
+  preText?: string[] | React.ReactNode[] | React.ReactNode;
+  postText?: string[] | React.ReactNode[] | React.ReactNode;
+  multiline?: boolean;
+  rows?: number;
+};
+
+const InternalInput = forwardRef<
```

Comment:
Change looks good but what's the intended difference between an InternalInput and a ChatInput? Or is this one Internal because it never gets exported? We'd just use ChatInput when we need an input (aside from the usage in this FormField component)?